coverage.py

```diff
@@ -242,16 +256,11 @@ def instantiate(cls, add_to, provider_kwargs, **kwargs):
         # the Overdrive API. We don't enforce that the collection
         # is an Overdrive collection, because we want to allow
         # unauthenticated lookups in the 'unaffiliated' collection.
-        try:
-            overdrive = instantiate(
-                OverdriveBibliographicCoverageProvider, providers,
-                provider_kwargs, collection=self.collection,
-                viaf=self.viaf, replacement_policy=self.replacement_policy
-            )
-        except CannotLoadConfiguration, e:
-            # No Overdrive collection is configured -- we can't
-            # handle Overdrive lookups, as much as we'd like to.
-            pass
+        overdrive = instantiate(
+            OverdriveBibliographicCoverageProvider, providers,
+            provider_kwargs, collection=self.collection,
+            viaf=self.viaf, replacement_policy=self.replacement_policy
+        )
```

Comment:
This stuff is already in master; I don't know why it's showing up in the diff here.